components/recurring-contributions/RecurringContributionsCard.js

```diff
@@ -47,7 +47,7 @@ const RecurringContributionsCard = ({
   const { formatMessage } = useIntl();
   const isError = status === ORDER_STATUS.ERROR;
   const isRejected = status === ORDER_STATUS.REJECTED;
-  const isActive = [ORDER_STATUS.ACTIVE, ORDER_STATUS.PROCESSING].includes(status) || isError;
+  const isActive = [ORDER_STATUS.ACTIVE, ORDER_STATUS.PROCESSING, ORDER_STATUS.NEW].includes(status) || isError;
```

Comment:
Strange to me that we consider a "New" order as active, since it hasn't been activated yet. Maybe we could figure out a better variable name:

```suggestion
  const isEditable = [ORDER_STATUS.ACTIVE, ORDER_STATUS.PROCESSING, ORDER_STATUS.NEW].includes(status) || isError;
```

Reply:
Yep, the statuses for recurring contributions are a bit off. I agree that `isEditable` is a better representation here for this context.

Reply:
Thanks. 👍🏽